src/core/lib/surface/server.c

```diff
@@ -55,7 +55,7 @@ typedef struct listener {
 
 typedef struct call_data call_data;
 typedef struct channel_data channel_data;
-typedef struct registered_method registered_method;
+typedef struct registered_method_t registered_method_t;
```

Comment:
See my comment above about not using the `_t` suffix.

Reply:
Same thing here

Reply:
Let's instead rename the field on line 79 from `registered_method` to `method`.

Reply:
done